__tests__/src/services/DataService.test.ts

```diff
@@ -137,22 +137,24 @@ describe('DataService', () => {
         team_members: { first_name: mockTeamMember1.first_name },
         service_category: ticket.id,
         label: 'mock app constant label',
-      }))
+      }));
       mockSupabaseClient.setTestData(expectedQueryResults);
       const expectedServiceRequestSummary = expectedQueryResults.map((data) => {
         const {
+          // eslint-disable-next-line @typescript-eslint/naming-convention
           clients, pets, team_members, id, label, ...ticket
         } = data;
         return {
-          id: id,
+          id,
```

Comment:
I saw sharrief did this as well in his PR. Is this needed?

Reply:
I am not sure, but it was part of a linting errors that my build and pr did not pass so, I did it, if he did it will just overwrite each other.